source/_components/notify.facebook.markdown

```diff
@@ -22,17 +22,29 @@ notify:
   - name: NOTIFIER_NAME
     platform: facebook
     page_access_token: FACEBOOK_PAGE_ACCESS_TOKEN
+    allowed_chat_ids:
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.